api/v2_search_filters.go

```diff
@@ -54,3 +58,60 @@ type TimeFilter struct {
 	StartTime *time.Time `json:"startTime,omitempty"`
 	EndTime   *time.Time `json:"endTime,omitempty"`
 }
+
+type SearchResponse interface {
+	GetDataLength() int
+}
+
+type SearchableFilter interface {
+	GetTimeFilter() *TimeFilter
+	SetStartTime(*time.Time)
+	SetEndTime(*time.Time)
+}
+
+// V2ApiMaxSearchHistory defines the maximum number of days in the past api v2 allows to be searched
+const V2ApiMaxSearchHistory = 92 //days
```

Comment:
lets add "days" in the variable name rather than a comment
```suggestion
const V2MaxSearchHistoryDays = 92 //days
```
something like that.